joint_state_controller:
  type: "joint_state_controller/JointStateController"
  publish_rate: 50

diff_drive_controller:
  type: "diff_drive_controller/DiffDriveController"
  left_wheel: left_wheel_joint
  right_wheel: right_wheel_joint

  # Wheel separation and diameter. These are both optional.
  # diff_drive_controller will attempt to read either one or both from the
  # URDF if not specified as a parameter
  wheel_radius: 0.024
  wheel_separation : 0.0925
  pose_covariance_diagonal:  [0.001, 0.001, 1000000.0, 1000000.0, 1000000.0, 1000.0]
  twist_covariance_diagonal: [0.001, 0.001, 1000000.0, 1000000.0, 1000000.0, 1000.0]

  cmd_vel_timeout: 1.0 #default 0.5

  # "enable_odom_tf" and "*_frame_id" are defined in raspimouse_control.launch
  # enable_odom_tf: true
  # base_frame_id: base_footprint
  # odom_frame_id: odom

  # Velocity and acceleration limits
  # Whenever a min_* is unspecified, default to -max_*
  linear:
    x:
      has_velocity_limits    : false
      max_velocity           : 2.0  # m/s
      has_acceleration_limits: true
      max_acceleration       : 3.0  # m/s^2
  angular:
    z:
      has_velocity_limits    : false
      max_velocity           : 8.0  # rad/s
      has_acceleration_limits: false
      max_acceleration       : 12.0  # rad/s^2